### ISSUE TITLE
Navigation menu icons are missing

### ISSUE THREAD

**Issue by breun** [apache/logging-log4j-kotlin#35]
The icons in the menu on https://logging.apache.org/log4j/kotlin/ don’t load.

![Scherm_afbeelding 2023-02-13 om 18 07 17](https://user-images.githubusercontent.com/102921/218665314-32f6e878-a14b-432b-808c-799f4e1a3d37.png)

**Comment by jvz** [comment]
Yes, I noticed that and fixed it in master. Will be addressed when we release 1.3.0. Looks to be some copy pasta errors with referencing included icon files.
